### PR TITLE
test: Update filter docker tests

### PR DESCRIPTION
**Pull request by nitkon**
Add additional tests to filter list.

Fixes: #1564

Signed-off-by: Nitesh Konkar <niteshkonkar@in.ibm.com>

### CODE REVIEW

**Comment by nitkon** [comment]
I am seeing a [check dmesg logs error](http://jenkins.katacontainers.io/job/kata-containers-proxy-Power8-16.04-PR-initrd/128/console) failure as well in the CI. Kindly DNM until I verify it. 
```
18:28:28     should be empty [It]
18:28:28     /home/jenkins/workspace/kata-containers-proxy-Power8-16.04-PR-initrd/go/src/github.com/kata-containers/tests/integration/docker/run_test.go:296
18:28:28 
18:28:28     Expected
18:28:28         <string>: [    0.000000] serial_ir serial_ir.0: Bad irq number or handler
18:28:28         [    0.000000] sir_ir: IRQ 4 already in use.
18:28:28     to be empty
```

**Comment by nitkon** [comment]
Updated my PR.
```
------------------------------

Ran 209 of 249 Specs in 2401.072 seconds
SUCCESS! -- 209 Passed | 0 Failed | 0 Pending | 40 Skipped PASS
```

/retest

File: .ci/ppc64le/configuration_ppc64le.yaml [hunk 1]
```diff
@@ -19,6 +19,12 @@ test:
 docker:
   Describe:
     - Update CPU set
+    - CPUs and CPU set
```
Comment (devimc) [inline .ci/ppc64le/configuration_ppc64le.yaml]: @nitkon did you file an issue for all thesee tests?

Comment (nitkon) [reply .ci/ppc64le/configuration_ppc64le.yaml]: @devimc : Issue for all these tests? umm..no. Should I for each test skipped?

Comment (devimc) [reply .ci/ppc64le/configuration_ppc64le.yaml]: @nitkon no, just one is fine, I would like to track what tests are not working in each architecture, also if you want you can add it to the limitation document

**Comment by nitkon** [comment]
@devimc: ARM + SLES CI's are stuck. Rest of the CI's are green.
I have updated this issue with the tests that I am skipping for ppc64le.

**Comment by devimc** [comment]
@nitkon 

> I have updated this issue with the tests that I am skipping for ppc64le.

once this pr is merged, this issue will be closed, you should file a new one to track the test that are failing on ppc

**Comment by nitkon** [comment]
@devimc : I created an [issue](https://github.com/kata-containers/tests/issues/1580) to track all the failing tests on ppc64le.

**Comment by nitkon** [comment]
@grahamwhaley : Good to merge?

File: .ci/ppc64le/configuration_ppc64le.yaml (hunk 1 above)
Comment (grahamwhaley) [reply .ci/ppc64le/configuration_ppc64le.yaml]: @nitkon - if you could add a comment with a reference to the Issue you opened in here, that'd be great, as then later on we can find/fix easier?

**Comment by nitkon** [comment]
All green!

File: .ci/ppc64le/configuration_ppc64le.yaml (hunk 1 above)
Comment (grahamwhaley) [reply .ci/ppc64le/configuration_ppc64le.yaml]: OK, #nnnn in comment is good enough I think - will merge.

**Pull request merged**